### ISSUE TITLE
RetinaNet-WeightNet can not convergence ，with loss = 1.72

### ISSUE THREAD

**Issue by convolutionROC** [megvii-model/WeightNet#4]
Though warming up is adopted, it also can't  convergence.

**Comment by nmaac** [comment]
Hi @convolutionROC , in our experiments it **converges well** without using warm up schedule.  To deal with your issue, we need detailed information:

* what is your dataset? what is your backbone model? did you use the ImageNet pre-trained backbone or from scratch?
* we suggest to use our code & models directly as your backbone, if not, please locate and provide the **minimum code**  added to your standard CNN baseline which causes the non-convergence problem (which means it converges if you remove the minimum code).
